src/main/scala/de/htwg/se/sudoku/model/fileIoComponent/fileIoMongoDBImpl/FileIO.scala

```diff
@@ -0,0 +1,80 @@
+package de.htwg.se.sudoku.model.fileIoComponent.fileIoMongoDBImpl
+
+import com.google.inject.{Guice, Inject}
+import com.google.inject.name.{Named, Names}
+import de.htwg.se.sudoku.SudokuModule
+import de.htwg.se.sudoku.model.fileIoComponent.FileIOInterface
+import de.htwg.se.sudoku.model.gridComponent.GridInterface
+import net.codingwell.scalaguice.InjectorExtensions.ScalaInjector
+import org.mongodb.scala.model.Projections
+import org.mongodb.scala.{Document, MongoClient, MongoDatabase}
+import play.api.libs.json.{JsValue, Json}
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+import scala.util.Try
+
+class FileIO @Inject()(@Named("MongoDBHost") host: String, @Named("MongoDBPort") port: Int) extends FileIOInterface {
+
+  private val mongoClient: MongoClient = MongoClient(s"mongodb://$host:$port")
+  private val db: MongoDatabase = mongoClient.getDatabase("sudoku-in-scala")
+  private val collection = db.getCollection("sudoku-in-scala-col")
+
+  override def load: Try[Option[GridInterface]] = {
+    val resF = collection.find().projection(Projections.excludeId()).toFuture()
+    val res = Await.result(resF, Duration.Inf)
```

Comment:
Maybe rename `res` to `result`? I don't think `res` is very meaningful. The same with `resF`.